plugin/storage/es/options_test.go

```diff
@@ -103,6 +106,19 @@ func TestOptionsWithFlags(t *testing.T) {
 	assert.True(t, primary.UseILM)
 }
 
+func TestEmptyRemoteReadClusters(t *testing.T) {
+	opts := NewOptions("es", "es.aux")
+	v, command := config.Viperize(opts.AddFlags)
+	err := command.ParseFlags([]string{
+		"--es.remote-read-clusters=",
+	})
+	require.NoError(t, err)
+	opts.InitFromViper(v)
+
+	primary := opts.GetPrimary()
+	assert.Equal(t, []string{}, primary.RemoteReadClusters)
```

Comment:
I think the following would be more expressive/readable: `assert.Empty(t, primary.RemoteReadClusters)`